scripts/update-website

```diff
@@ -20,7 +22,7 @@ cp -r docs/output/* "$tmpdir"
 cp .gitignore "$tmpdir"
 
 # copy to gh-pages
-git checkout gh-pages
+git switch gh-pages
```

Comment:
I'd argue against this. As of `Git 2.40.1`, git-switch is experimental and thus subject to change when `git` changes. Since you're using `ubuntu-latest`, you don't have any guarantees about the version of `git` and thus the behavior of this command may change randomly at some unexpected time.

Reply:
Reverted, thanks.